benchmarks/cpp/nvfuser/gelu_backward.cpp

```diff
@@ -23,19 +23,19 @@ static void setupFusion(Fusion* fusion) {
   const float k_010 = 0.1070322243;
 
   // gradient tensor
-  auto t0 = TensorViewBuilder().ndims(3).dtype(DataType::Half).build();
+  auto t0 = makeContigTensor(3, DataType::Half);
```

Comment:
Since the contiguity wasn't originally specified, should this be `makeSymbolicTensor`?

```
// Make a tensor that is known to be non-contiguous of dimensionality=ndims,
// but unknown sizes
TensorView* makeSymbolicTensor(size_t ndims, DataType dtype = DataType::Float) {
  return TensorViewBuilder().ndims(ndims).dtype(dtype).build();
}
```